pkg/download/classic/download.go

```diff
@@ -141,9 +141,6 @@ type value struct {
 }
 
 func (v value) id() string {
-	if v.value.Id == v.parentConfigId {
```

Comment:
Did you test if dashboard-share-settings are correctly created after that change @warber?

I think this might have been added before the introduction of creating an API instance with the parent's ID pre-coded into the API path (`pkg/api/api.go:98`). We might have coded the ID into the URL via the config ID at the start and this ensures we have the right ID available? @arthurpitman does that maybe sound familiar as a reason to introduce this code?

Reply:
@UnseenWizzard yes, it works 🤷